data-raw/nv_neuro.R

```diff
@@ -0,0 +1,215 @@
+# Dataset: nv_neuro
+# Description: Create NV test SDTM dataset for Alzheimer's Disease (neuro studies)
+
+# Load libraries ----
+
+library(tibble)
+library(dplyr)
+library(stringr)
+library(admiral)
+
+# Read input data ----
+
+data("dm_neuro")
+
```

Comment:
Could you please limit to one blank line max across the program - thanks

Reply:
Done.